cdap-ui/app/cdap/components/PaidBanner/PaidBanner.scss

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+@import "../../styles/variables.scss";
+
+.paid-banner {
+  width: 30px;
+  display: inline-block;
+  position: relative;
+  color: white;
```

Comment:
Use color from `colors.scss`, here and line 45

Reply:
done